ci/docker-compose.yml

```diff
@@ -14,6 +14,33 @@ services:
       timeout: 5s
       retries: 5
 
+  zookeeper:
+    image: confluentinc/cp-zookeeper
+    ports:
+      - "2181:2181"
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
+      ZOOKEEPER_SYNC_LIMIT: 2
+
+  kafka:
```

Comment:
Running Kafka here might lead to high network fee. Better to use a hosted Kafka service from AWS.

Reply:
Just ensure Kafka is running in the same region as RisingWave Cloud, then everything should be fine.

Reply:
All services and processes of the automated benchmark are in the same region, so there should be no excessive network charges.